easybuild/easyconfigs/n/netCDF/netCDF-4.5.0-foss-2018a.eb

```diff
@@ -0,0 +1,36 @@
+name = 'netCDF'
+version = '4.5.0'
```

Comment:
@molden We should go forward with `netCDF` 4.6.0 for the `2018a` toolchains? Cfr. #6012

Reply:
@boegel ok 4.6.0 is ok for me, you can close this PR.